### ISSUE TITLE
Erro

### ISSUE THREAD

**Issue by joelhenriquedev** [LukeDetonator3000/Pokemon#2]
@LukeDetonator3000 Está com erro na descrição da carta, a primeira carta tem descrição, e a segunda e terceira não, ai a quarta tem descrição e a quinta e sexta não. Veja isso ai.

**Comment by Nizekul** [comment]
ok, no caso é para colocar o que?

**Comment by joelhenriquedev** [comment]
no lugar de undefined tem que colocar a descrição

**Comment by Nizekul** [comment]
eles tem uma definição já pronta?

**Comment by joelhenriquedev** [comment]
defina definição.

**Comment by Nizekul** [comment]
uma descrição pronta igual as outras, que eu nao vou precisar digitar cada 1

**Comment by joelhenriquedev** [comment]
sim, todas cartas tem descrições. Tem algum erro no seu código que está mostrando essa descrição para apenas algumas cartas, outras estão recebendo undefined.
